modules/OCPP/display_message/display_messageImpl.cpp

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
the whole display_message directory shall be removed from OCPP. Its not something that the module provides, but optionally requires (see the manifest)